samples/04_gis_analysts_data_scientists/forecasting_air_temperature_in_california.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>combine the above two lines in the same cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>